lib/hardware_services/network_drivers/hilapi.py

```diff
@@ -0,0 +1,107 @@
+# this is a library to conveniently do REST calls to HIL server
+
+import requests
+import sys
+import yaml
+
+def error_check(response):
```

Comment:
This is not the inventory or network driver class. They are in separate files.

You might want to check how Isaac incorporated this code into the inventory HIL driver. I can do the same if you want. But I think HIL RESTful are logically unrelated to the HIL driver classes.